package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3296,42 +3443,19 @@ def _init_bonds(self):
            Now only accepts list of tuples as input, previously accepted either
```

Comment:
The "previously accepted" doc seems irrelevant; if at all put in a versionchanged markup.